fast/stages/2-networking-c-nva/data/firewall-rules/heartbeat/rules.yaml

```diff
@@ -0,0 +1,22 @@
+# skip boilerplate check
+---
+# Terraform will be unable to decode this file if it does not contain valid YAML
+# You can retain `---` (start of the document) to indicate an empty document.
+
+ingress:
+  allow-hc-nva-ssh-landing:
```

Comment:
firewall rules must be globally unique, this rule name is used in three places:
* ./data/firewall-rules/management/default-ingress.yaml
* ./data/firewall-rules/landing/default-ingress.yaml
* ./data/firewall-rules/heartbeat/default-ingress.yaml